generate.sh

```diff
@@ -15,6 +15,7 @@ set -euo pipefail
 PACS=(
     "lpc11uxx:--nightly"
     "lpc176x5x:--nightly"
+    "lpc11u6x:--nightly"
```

Comment:
Here's another thing that jumps out to me. I think the `--nightly` passes this flag to svd2rust when generating the crate. My questions:
- Is this necessary?
- Is this even a thing anymore? I'm not totally up to date on svd2rust, but I believe this might have been removed in the latest versions.

@dbrgn @rnestler Thoughts?

Reply:
Seems to be. It only generated untagged union code for overlapping registers if the flag is passed down.

Reply:
Yep, the NXP register maps require untagged unions, which are nightly-only.